Model/Service/EmailSend.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Flancer32\Csp\Model\Service;
+
+use Magento\Framework\App\Helper\Context;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\Mail\Template\TransportBuilder;
+use Magento\Framework\Translate\Inline\StateInterface;
+use Magento\Framework\Escaper;
+
+class EmailSend
+{
+    /**
+     * @var TransportBuilder
+     */
+    private $transportBuilder;
+    /**
+     * @var StateInterface
+     */
+    private $inlineTranslation;
+    /**
+     * @var ScopeConfigInterface
+     */
+    private $scopeConfig;
+    /**
+     * @var Escaper
+     */
+    private $escaper;
+    /**
+     * @var \Psr\Log\LoggerInterface
+     */
+    private $logger;
+
+    public function __construct(
+        TransportBuilder $transportBuilder,
+        StateInterface $inlineTranslation,
+        ScopeConfigInterface $scopeConfig,
+        Escaper $escaper,
+        Context $context)
+    {
+        $this->transportBuilder = $transportBuilder;
+        $this->inlineTranslation = $inlineTranslation;
+        $this->scopeConfig = $scopeConfig;
+        $this->escaper = $escaper;
+        $this->logger = $context->getLogger();
+    }
+
+    /**
+     * @param $reportHtml
+     * @return array
+     * @throws EmailSendException
+     */
+    public function with($reportHtml)
```

Comment:
`emailSend->with(html)` reads nice, `sendEmail->with(html)` would be even better :smiley: